substrate-node/pallets/pallet-smart-contract/src/tests.rs

```diff
@@ -4072,7 +4115,7 @@ fn validate_distribution_rewards(
     // 5% is sent to the staking pool account
     assert_eq_error_rate!(
         staking_pool_account_balance,
```

Comment:
In all below, prefer using:
~~~rust
    assert_eq!(
        foundation_account_balance - 500,
        Perbill::from_percent(10) * total_amount_billed
    );
~~~
to be aligned with previous `fn test_node_contract_billing_details()` test.
Or stay like this but modify `fn test_node_contract_billing_details()` instead.